README.md

```diff
@@ -118,18 +151,52 @@ const List<Map<String, dynamic>> countries = [
 ]
 ```
 
-### Device Region code
+#### Device Region code
 
 It is possible to fetch the region code from the device. This will give you the two letter country code. (e.g. US, UK, ...)
 
 ```dart
 String code = await plugin.carrierRegionCode();
 ```
 
-## Contributors
+## Support
+
+### Contribute
+
+You have some ways to contribute to this packages
+
+ - Beginner: Reporting bugs or request new features
+ - Intermediate: Implement new features (from issues or not) and created pull requests
+ - Advanced: Join to [organization](#ecosystem) like a member and help coding, manage issues, dicuss new features and other things
+
+ See [contribution file](https://github.com/flutter-form-builder-ecosystem/.github/blob/main/CONTRIBUTING.md) for more details
+
+### Questions and answers
+
+You can join to [our Discord server](https://discord.gg/25KNPMJQf2)
+
+### Donations
+
+Donate or become a sponsor of Flutter Form Builder Ecosystem
+
+[![Become a Sponsor](https://opencollective.com/flutter-form-builder-ecosystem/tiers/sponsor.svg?avatarHeight=56)](https://opencollective.com/flutter-form-builder-ecosystem)
+
+## Roadmap
+
+- [Simplify example code](https://github.com/flutter-form-builder-ecosystem/phone_number/issues/79)
+- [Implement integration tests](https://github.com/flutter-form-builder-ecosystem/phone_number/issues/74)
+- [Solve open issues](https://github.com/flutter-form-builder-ecosystem/phone_number/issues), [prioritizing bugs](https://github.com/flutter-form-builder-ecosystem/phone_number/labels/bug)
```

Comment:
"Fix issues" will be good IMO

Reply:
Yeah, but I mean that we can close the issues, be it bugs, features or other things. _Fix_ I only use for issues.
Maybe has a better word than solve and fix